### PR TITLE
Create apache-spark-1.6.1.conf

### PR DESCRIPTION
**Pull request by pescerosso**
Adding a package for Spark 1.6.1, got the code from Dean's blog https://www.apcera.com/blog/apache-spark-apcera-platform-community-edition

### CODE REVIEW

**Comment by variadico** [comment]
LGTM

**Comment by zquestz** [comment]
LGTM

**Comment by earlruby** [comment]
LGTM

File: packages/apache-spark-1.6.1.conf [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name:    "spark"
+version: "1.6.1"
+
+sources [
+  {url: "http://apache.mirror.anlx.net/spark/spark-1.6.1/spark-1.6.1-bin-hadoop2.6.tgz"},
```
Comment (earlruby) [inline packages/apache-spark-1.6.1.conf]: * Needs a SHA256 signature.
* `spark-1.6.1-bin-hadoop2.6.tgz` should be added to our S3 repo.

**Comment by earlruby** [comment]
`README.md` and `Verification.md` should also be updated.

File: packages/apache-spark-1.6.1.conf [hunk 2]
```diff
@@ -0,0 +1,31 @@
+name:    "spark"
```
Comment (earlruby) [inline packages/apache-spark-1.6.1.conf]: `name

**Comment by earlruby** [comment]
I asked @pescerosso if I could make these changes and he said yes, so I did:

* Added a SHA256 signature.
* Added `spark-1.6.1-bin-hadoop2.6.tgz` to our S3 repo.
* Updated `README.md` and `Verification.md`.

**Comment by earlruby** [comment]
LGTM

**Comment by variadico** [comment]
LGTM the new edits.

**Pull request merged**